roles/custom/matrix-synapse/defaults/main.yml

```
@@ -334,6 +334,11 @@ matrix_synapse_limit_profile_requests_to_users_who_share_rooms: false
 # Defaults to 'true'.
 matrix_synapse_include_profile_data_on_invite: true
 
+
+# User search behaviour
+matrix_synapse_search_all_users: false
+matrix_synapse_prefer_local_users: false
```

Given which actual settings these variables control, to keep consistency with the rest of the playbook, these variables should be renamed:

```suggestion
matrix_synapse_user_directory_search_all_users: false
matrix_synapse_user_directory_prefer_local_users: false
```